packages/shared/src/components/ProfileMenu.tsx

```diff
@@ -58,6 +72,21 @@ export default function ProfileMenu(): ReactElement {
           </a>
         </Link>
       </Item>
+      {showSearchReferral && (
+        <Item>
+          <button
+            type="button"
+            className="flex items-center min-w-[12.5rem]"
```

Comment:
The min-width feels odd, why is it needed?

Reply:
Also why is it a normal html button?
Our own one would allow left and right icons right?

Reply:
This is a button in the profile menu item. It would be very enlarged for the context menu.